bug in unbacked_bindings for a*u0

pianpwk opened the pull request:
Summary: we were storing a*u0 instead of u0 in unbacked_bindings / unbacked_var_to_val

Test Plan: -

Differential Revision: D64508936

pianpwk committed:
bug in unbacked_bindings for a*u0

Summary: we were storing a*u0 instead of u0 in unbacked_bindings / unbacked_var_to_val

Test Plan: -

Differential Revision: D64508936

ezyang commented:
Needs test

ezyang reviewed (approved):
Logic change looks correct

pianpwk committed:
bug in unbacked_bindings for a*u0 (#138136)

Summary:

we were storing a*u0 instead of u0 in unbacked_bindings / unbacked_var_to_val

Test Plan: -

Reviewed By: ezyang

Differential Revision: D64508936

pianpwk committed:
Merge branch 'export-D64508936' of https://github.com/pytorch/pytorch into export-D64508936

pianpwk committed:
export
